modules/installation-requirements-user-infra.adoc

```diff
@@ -230,6 +230,88 @@ On your z/VM instances, set up:
 * 16 GB for the temporary {product-title} bootstrap machine
 endif::ibm-z[]
 
+ifdef::ibm-power[]
+
+[id="minimum-ibm-power-system-requirements_{context}"]
+== Minimum IBM Power Systems requirements
+
+You can install {product-title} version {product-version} on the following IBM hardware:
+
+* IBM POWER8 or POWER9 processor-based systems
+
+[discrete]
+=== Hardware requirements
+
+* 6 IBM Power bare metal servers or 6 LPARs across multiple PowerVM servers.
```

Comment:
Some of these bullets have periods and others don't. Suggest going through and removing all of them in the PR (since they're not full sentences)